### DISCUSSION TITLE
Confirmation pop up on save button

### DISCUSSION THREAD

**Discussion by niko4299** [argoproj/argo-cd#13868] [Q&A]
Hey!

Currently, when you change something in your config map and press save you will refresh pods (for example. number of replicas) which is a great thing but maybe you don't want to do that in some environments. In my project, I have two environments one used by customers and one used for testing. Is there a way to add some kind of popup button that asks for confirmation when you click save so I can know okay this is not env used by customers?

Thank you.

**Comment by carslen** [comment]
You're talking about applying changes using ArgoCD WebUI to k8s resources, right?

I guess, when acting like this, then you must be aware of what you're doing. I would not expect to receive a popup like "Are you shure?". To apply possible critical changes (to the correct environment) you should commit your changes to your repository + having a proper review process. Follow the GitOps approach instead of applying changes via editing resources via WebUI.

**Reply by niko4299** [reply]
Right but sometimes when I want to test some env variable it is much easier to change it through WebUI than go through all the processes with the GitHub repository. The environments look the same and I just want to have one more safety check to prevent something bad from happening. I saw that you can change the colors of your envs so I thought maybe I can somehow add this popup too.

**Reply by carslen** [reply]
I understand your pain and I'm doing the same on my systems every now and then. But if you do, you should be careful to modify the correct one :-)
